contracts/src/AttestationRegistry.sol

```diff
@@ -296,4 +316,23 @@ contract AttestationRegistry is OwnableUpgradeable {
     }
     return result;
   }
+
+  /**
+   * @notice Generate an attestation ID, prefixed by the Verax chain identifier
+   * @param id The attestation ID (coming after the chain prefix)
+   * @return The attestation ID
+   */
+  function generateAttestationId(uint256 id) public view returns (bytes32) {
```

Comment:
@alainncls could this function be internal maybe ?

Reply:
I feel like having the chain prefix as a bytes32 as oppose to uint256, then we can simply do:

bytes32(abi.encode(prefix+ counter))

Reply:
Done ✅ 
It's still tested and used in tests, via a "harness contract" (cf. [Foundry doc](https://book.getfoundry.sh/tutorials/best-practices#internal-functions)).